engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql

```diff
@@ -833,3 +833,7 @@ INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_vers
 
 -- Fix OS category for Guest OS 'Other PV Virtio-SCSI (64-bit)'
 UPDATE `cloud`.`guest_os` SET category_id = 7 WHERE id = 275 AND display_name = 'Other PV Virtio-SCSI (64-bit)';
+
+ALTER TABLE `cloud`.`user_vm` ADD COLUMN `user_vm_type` varchar(255) DEFAULT "UserVM" COMMENT 'Defines the type of UserVM';
```

Comment:
@Pearl1594 is this not redundant of _Type_ column in _vm_instance_ table? This _type_ column already used to persist different system VMs  (ConsoleProxy, SecondaryStorageVm, DomainRouter), may be you can use this for different user VMs as well.

Reply:
Intention of adding a user vm type - as a sub-type of VMInstance type - User is to basically reduce any regression - changing User to other type to accommodate CKS / K8s nodes would possibly require a lot more changes

Reply:
Ok @Pearl1594 I think it is better to use single source for defining the instance type.